torch/distributed/run.py

```diff
@@ -729,12 +744,12 @@ def config_from_args(args) -> Tuple[LaunchConfig, Union[Callable, str], List[str
         else:
             if args.module:
                 raise ValueError(
-                    "Don't use both the '--no_python' flag"
+                    "Don't use both the '--no-python' flag"
                     " and the '--module' flag at the same time."
                 )
             cmd = args.training_script
     if not use_env:
-        cmd_args.append(f"--local_rank={macros.local_rank}")
```

Comment:
Hi! This change broke our code and probably not only ours. Can we still have an option to use `--local_rank` with an underscore?

Reply:
sure, can you send us a patch